resolwe/flow/executors/startup_processing_container.py

```diff
@@ -624,8 +618,8 @@ def start_processing_container():
     JSON_LOG_PATH.touch()
 
     # Replace the code bellow with asyncio.run when we no longer have to
-    # support python 3.6.
+    # support python 3.4+.
```

Comment:
I prefer to keep `3.6` here as `async.run` was added in 3.7, so we can start using it once we drop support for 3.6.